xbmc/cores/VideoPlayer/VideoRenderers/DebugRenderer.h

```diff
@@ -12,14 +12,21 @@
 
 #include <string>
 
+#define NUM_ELE 6
+
+struct SInfo
+{
+  std::string s[NUM_ELE];
```

Comment:
Same comment as above, a variable named "s" is a bit terse. Can it be extended? Using a short "s" could also be desirable. In that case, documenting what "s" stands for would be a good idea.

Reply:
Changed to "line"